qcog_python_client/qcog/pytorch/validate/validate_utils.py

```diff
@@ -6,24 +6,27 @@
 import sys
 from functools import lru_cache
 
+from qcog_python_client.log import qcoglogger as logger
```

Comment:
So just as a reference the best practice here is to actually call get logger in each module as that allows better granularity for the logging configuration.

So this would be like in the main `__init__.py` the `log.py` is invoked for configuring the logging library globally and then in each file you would do `logging.getLogger(__name__)`

https://docs.python.org/3/howto/logging.html#when-to-use-logging for some examples. Maybe you want to make a small ticket to add this to some remediation later, this logging fix should be fast for another PR.

Reply:
Ticket added  https://app.asana.com/0/1207592301203555/1208134743848835